sherpa/conftest.py

```diff
@@ -64,7 +64,14 @@ def pytest_addoption(parser):
             r"File '/data/regression_test/master/in/sherpa/aref_Cedge.fits' does not have write permission.  Changing to read-only mode."
         ],
     RuntimeWarning:
-        [r"invalid value encountered in sqrt", ],
+        [r"invalid value encountered in sqrt",
+         # The following is added for the test_can_use_swift_data
+         # test in sherpa/astro/tests/test_astro_data_swift_unit.py;
+         # as the test checks this warning is created it would be useful
+         # if the test could eitehr clear the warnings, or tell this
+         # fixture to ignore it, rather than making it a global flag
+         r"divide by zero encountered in divide",
```

Comment:
@DougBurke any comments on my findings? I'd try running CI with the warnings removed from the whitelist. It bothers me that there isn't anything in the pytest docs explaining why the warnings wouldn't bubble up after you capture them with pytest itself, but then again maybe I did something wrong and Travis wouldn't be happy. This is the only (minor) issue preventing me from merging this PR.